mkdocs.yml

```diff
@@ -40,6 +40,9 @@ theme:
     custom_dir: custom_theme/
 copyright: Ambianic.ai &copy; 2019-2021
 docs_dir: docs-md
+
```

Comment:
@vickywane the new premium services page needs to be linked in the main menu in this mkdocs.yml file. Mayb under 
```
- User Guide:
  ...
  - Home Assistant Integration: users/hass-integration.md
  - Premium Services users/premium-services.md 
```

Reply:
Alright. 

I have made this adjustment.